src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -169,12 +162,17 @@ export default function StatusCard({ sightingData }) {
     });
   }
 
+  const formattedIdentificationStartTime = identificationStartTime
+    ? formatDateCustom(identificationStartTime, 'PPp')
+    : '';
+
   return (
     <Card titleId="IDENTIFICATION_PIPELINE_STATUS" maxHeight={900}>
       <Timeline>
         <TimelineStep
           Icon={ReportIcon}
           titleId="SIGHTING_SUBMISSION"
+          isUserInTheLoop={false}
```

Comment:
Love this name lol, although I can't quite tell without digging deeper why all it dictates is the difference between ProgressIcon and CircularProgressAsIcon. 

In other words, what is the point of this prop?

Reply:
This is no longer relevant because the same icon is being used for all steps again.